client/my-sites/checkout/composite-checkout/hooks/use-show-add-coupon-success-message.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+import { useEffect, useRef } from 'react';
+
+export default function useShowAddCouponSuccessMessage(
```

Comment:
I can't find any place using this hook. Maybe it has been deprecated at some points? or maybe I'm simply overlooking?